pino-eventhub.js

```diff
@@ -67,19 +74,44 @@ function pinoEventHub (opts) {
     }
   }
 
+  const index = opts.index || 'pino'
+  const type = opts.type || 'log'
+
   const writable = new Writable({
     objectMode: true,
     highWaterMark: opts['bulk-size'] || 500,
-    write: function (body, enc, done) {
-      debug(`write: body =`, body)
-      debug(`write: typeof ===`, typeof body)
-      if (body) {
-        var req = https.request(options, callback(done))
+    writev: function (lines, done) {
+      // https://docs.microsoft.com/en-us/rest/api/eventhub/send-batch-events
+      const events = lines
+        .map(line => {
+          // check if console output is a string or object
+          var parsed = new Parse(line)
+          if (parsed.err) {
+            return `{"Body":"${line}"}`
```

Comment:
It's better if you do a JSON.stringify here, otherwise you might get into escaping issues